README.md

```diff
@@ -11,12 +11,14 @@
 
 ## Installation
 
-`ko` can be installed and upgraded by running:
+`ko` can be installed and upgraded from our [releases](https://github.com/ko/releases) by choosing
```

Comment:
Looks like the link is not right - should be https://github.com/google/ko/releases